samples/stock/demo/drag-on-axis/demo.js

```diff
@@ -0,0 +1,132 @@
+(function (Highcharts) {
+    // Configurable
+    const MOUSE_SENSITIVITY = 0.002,
+        SCROLL_SENSITIVITY = 0.0002;
+    function centerZoom(axis, strength) {
+        let zoomOffset = (axis.max - axis.min) / 2;
+        const center = axis.min + zoomOffset;
+
+        zoomOffset += strength * axis.toValue(1);
+
+        if (zoomOffset < 0) {
+            zoomOffset = 0;
+        }
+
+        axis.setExtremes(center - zoomOffset, center + zoomOffset, true, false);
+        if (axis.chart.resetZoomButton) {
+            axis.chart.resetZoomButton = axis.chart.resetZoomButton.destroy();
+        }
+        axis.chart.showResetZoom();
+    }
+
+    Highcharts.addEvent(Highcharts.Axis, 'afterInit', function () {
+        const axis = this,
+            renderer = axis.chart.renderer;
+
+        if (!axis.isXAxis && !axis.options.isInternal) {
+            // Create a transparent rectangle on yAxis to create a zoom area
+            axis.axisZoomRect = renderer.rect()
+                .attr({
+                    width: 50,
+                    height: 300,
```

Comment:
Static height? To cover charts of different sizes, including resizing of existing charts, I think the height should be dynamically set to `chart.plotHeight` or `yAxis.len` along with x and y on line 70.

Reply:
Oh snap, that's a prod leftover - fixed now, thanks 👍